### PR TITLE
pref: optimized nginx configuration

### PR DESCRIPTION
**Pull request by lbbniu**
#### What type of PR is this?
optimized nginx configuration

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: deploy/craned/deployment.yaml [hunk 1]
```diff
@@ -150,6 +150,17 @@ metadata:
   name: nginx-conf
   namespace: crane-system
 data:
+  common_params: |
```
Comment (qmhu) [inline deploy/craned/deployment.yaml:153]: How to make it take effect when you move to here?

Comment (lbbniu) [reply deploy/craned/deployment.yaml:153]: include conf.d/common_params;

**Review by qmhu — APPROVED** [review]
/LGTM

**Comment by lbbniu** [comment]
> /LGTM
Sync update
https://github.com/gocrane/helm-charts/pull/76

**Pull request merged**